charts/sn-platform/templates/openshift/_openshift.tpl

```diff
@@ -1,3 +1,12 @@
-{{- define "pulsar.openshift.scc.name" -}}
-{{ template "pulsar.namespace" . }}-{{ template "pulsar.fullname" . }}-openshift-scc
+{{- define "pulsar.openshift.scc" -}}
+{{- $defaultSCC := printf "%s-%s-openshift-scc" (include "pulsar.namespace" .) (include "pulsar.fullname" .) -}}
+{{- .Values.openshift.scc.name | default $defaultSCC }}
+{{- end -}}
+
+{{- define "pulsar.openshift.role" -}}
+{{- printf "%s-%s-openshift-role" (include "pulsar.namespace" .) (include "pulsar.fullname" .) }}
+{{- end -}}
+
+{{- define "pulsar.openshift.rolebinding" -}}
+{{- printf "%s-%s-openshift-rolebinding" (include "pulsar.namespace" .) (include "pulsar.fullname" .) }}
```

Comment:
The suffix `-rolebinding` seems to be redundant.

Reply:
Just keep consistent with similar naming convention. e.g, https://github.com/streamnative/charts/blob/722bffe84e208e5f15f968520da6dbad5d0c34dd/charts/sn-platform/templates/zookeeper/zookeeper-backup-rolebinding.yaml#L25